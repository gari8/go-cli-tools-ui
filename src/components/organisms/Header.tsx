import { Box } from "@chakra-ui/react";

const Header = () => {
  return <Box>aa</Box>;
};

export default Header;
