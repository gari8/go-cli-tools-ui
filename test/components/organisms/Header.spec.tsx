import { render } from "@testing-library/react";
import Header from "../../../src/components/organisms/Header";

describe("Header", (): void => {
  test("is components", () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
});
