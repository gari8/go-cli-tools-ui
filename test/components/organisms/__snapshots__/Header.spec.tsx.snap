// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header is components 1`] = `
<DocumentFragment>
  <div
    class="css-0"
  >
    aa
  </div>
</DocumentFragment>
`;
